rasgotransforms/rasgotransforms/transforms/histogram/histogram.sql

```diff
@@ -11,9 +11,19 @@ SELECT
   ,COUNT(1) AS REC_CT
 FROM
   (SELECT {{ column }}::float AS COL FROM {{ source_table }}
-  {%- if filter_statements is iterable -%}
-  {%- for filter_statement in filter_statements %}
-  {{ 'WHERE' if loop.first else 'AND' }} {{ filter_statement }}
+  {%- if filter_statements is defined and filter_statements %}
```

Comment:
is this a double existence check?

Reply:
Not a double existence - `is defined` is a presence check, and `filter_statements` is a check to see if the iterable contains anything